items.json

```diff
@@ -30522,7 +30219,7 @@
       },
       {
         "class": "TwoHandedPolearm",
-        "itemUsage": "polearm_block_long_shield_thrust",
+        "itemUsage": "polearm_bracing",

```

Comment:
By doing this you killed two handed for every polearm in the game. What was your plan?

Reply:
not sure how this changed...ill go back and redo this commit